ampproject/amp-wp
# Video poster image size improvements

Fixes #2881.

To test:

1. Get a demo video for testing, e.g. https://www.pexels.com/video/blue-sky-with-white-clouds-above-a-body-of-water-1860684/
2. Create a new page
3. Add background media
4. Upload video
5. Notice that poster image is extracted and saved
6. Save and preview story.
7. Poster file name should have suffix `-2560x1440.png`
